index.html

```diff
@@ -291,6 +303,63 @@
           </dt>
         </dl>
       </section>
+      <section>
+	<h2>
+	  Media Devices
+	</h2>
+	<p>
+	  The <code><dfn id='idl-def-PermissionName.camera'>camera</dfn></code>,
+	  <code><dfn id='idl-def-PermissionName.microphone'>microphone</dfn>
+	  </code>,
+	  and <code><dfn id='idl-def-PermissionName.speaker'>speaker</dfn>
+	  </code>
+	  permissions are associated with permission to use media
+	  devices as specified in [[GETUSERMEDIA]] and
+	  [[audio-output]].
+	</p>
+	<p>These three permissions have an associated PermissionDescriptor:
+	</p>
+	<dl class='idl' title='dictionary DevicePermissionDescriptor :
+	    PermissionDescriptor'>
+	  <dt>
+	    DOMString deviceId
+	  </dt>
+	</dl>
+	<p>A permission covers access to the device given in the
+	associated descriptor.
+	</p>
+	<p>
+	  If the descriptor does not have a deviceId, its semantic is that
+	  it queries for access to any device of that class.
+	</p>
+	<p>
+	  If the deviceId is the special string "*", it queries for
+	  access to all devices of that class, whether they exist
+	  at the moment or are attached at a later time. When such a
+	  permission has been granted, queries for permission to any
+	  specific device MUST return "granted" as long as the
+	  deviceId is the ID of a connected device of the given class.
```

Comment:
Hm... for the case where it's not '*' and it's not a legal deviceId?

No, that's not specified - I thought the API would have a general text saying "if the parameters are illegal, the result is an error" - but there doesn't seem to be any such language; when retrieving a permission object that doesn't exist, the result is implementation-defined, but always returning "prompt" is one suggested resolution.

I'd suggest an error for illegal descriptors. WDYT?